core/modules/taxonomy/views/views_handler_filter_term_depth.inc

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * @file
+ * Definition of views_handler_filter_term_depth.
+ */
+
+/**
+ * Filter handler for taxonomy terms with depth.
+ *
+ * This handler is actually part of the node table and has some restrictions,
+ * because it uses a subquery to find nodes with.
+ *
+ * @ingroup views_filter_handlers
+ */
+class views_handler_filter_term_depth extends views_handler_filter_many_to_one {
+  function operator_options($which = 'title') {
+    return array(
+    );
+  }
+
+  function option_definition() {
+    $options = parent::option_definition();
+
+    $options['value'] = array('default' => 0);
+
+    return $options;
+  }
+
+  function value_form(&$form, &$form_state) {
+    parent::value_form($form, $form_state);
+
+    $form['value'] = array(
+      '#type' => 'weight',
+      '#title' => t('Depth'),
+      '#default_value' => $this->options['depth'],
+      '#description' => t('The depth will match nodes tagged with terms in the hierarchy. For example, if you have the term "fruit" and a child term "apple", with a depth of 1 (or higher) then filtering for the term "fruit" will get nodes that are tagged with "apple" as well as "fruit". If negative, the reverse is true; searching for "apple" will also pick up nodes tagged with "fruit" if depth is -1 (or lower).'),
```

Comment:
...same as previous respective chunk of text: "content" instead of "nodes" and the comma should go before the "then".